cnudie/util.py

```diff
@@ -414,6 +416,121 @@ def find_changed_component(
     )
 
 
+def format_component_diff(
+    component_diff: ComponentDiff,
+    to_component_name: str,
+    delivery_dashboard_url: str | None
+):
+    if delivery_dashboard_url:
+        dashboard_view_diff = (
+            f'{delivery_dashboard_url}/#/component?name='
+            f'{to_component_name}&view=diff'
```

Comment:
Instead of only linking to the diff page, you could also already display the calculated diff by adding:
```
&componentDiff=<left-component-name>:<left-component-version>:<right-component-name>:<right-component-version>
```